schema_editor/app/scripts/views/geography/add-partial.html

```diff
@@ -0,0 +1,99 @@
+<div class="form-area">
+    <div class="col-sm-8 col-sm-offset-3">
+        <div class="form-area-heading">
+            <h2>Add New Geography</h2>
+            <div class="content-border"></div>
+        </div>
+        <div ng-switch="geoAdd.uploadState">
+            <div class="alert alert-info" role="alert" ng-switch-default>
+                <span class="sr-only">Info:</span>
+                Upload your zipped shapefile; once uploaded, select a primary field for display
+            </div>
+            <div class="alert alert-info" ng-switch-when="requesting">
+                <span class="sr-only">Loading...:</span>
+                Loading...
+            </div>
+            <div class="alert alert-danger" role="alert" ng-switch-when="upload-error">
+                <span class="glyphicon glyphicon-exclamation-sign" aria-hidden="true"></span>
+                <span class="sr-only">Error:</span>
+                {{ geoAdd.errorMessage }}
+            </div>
+            <div class="alert alert-success" role="alert" ng-switch-when="upload-success">
+                <span class="glyphicon glyphicon-ok" aria-hidden="true"></span>
+                <span class="sr-only">Success:</span>
+                Geography upload successful! Select a primary field below.
+            </div>
+            <div class="alert alert-success" role="alert" ng-switch-when="update-success">
+                <span class="glyphicon glyphicon-ok" aria-hidden="true"></span>
+                <span class="sr-only">Success:</span>
+                Geography update successful!
+            </div>
+        </div>
+        <form name="geoAddForm" ng-submit="" novalidate>
+            <div class="form-group col-sm-12">
+                <label for="geo-label">Geography Label</label>
+                <input type="text"
+                       name="label"
+                       ng-model="geoAdd.geoFields.label"
+                       class="form-control"
+                       id="geo-label"
+                       placeholder="Manila"
```

Comment:
Maybe change these Manila defaults to something else to avoid confusion, since it's more a collection of areas rather than a single area. Perhaps something like: "Pilot Areas", "Police Districts", or something along those lines.